maistra/run-ci.sh

```diff
@@ -14,6 +14,11 @@ time bazel build \
 echo "Build succeeded. Binary generated:"
 bazel-bin/source/exe/envoy-static --version
 
+# Build tests
+time bazel build \
+  ${COMMON_FLAGS} \
+  //test/...
```

Comment:
Perhaps add `--build_tests_only` too?

Reply:
I tried this change with local disk cache. It helps avoid a CPU usage spike. 
hm, I assume all files under the //test/ are tests. I can add --build_tests_only in that line